micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/JettyServerCustomizer.java

```diff
@@ -150,22 +163,88 @@ public void contextInitialized(ServletContextEvent sce) {
             servletContext.addServlet("WelcomeApp", new ServletContainer(new WelcomeApp())).addMapping("/api/welcome/*");
             registerFilter("ProcessEnginesFilter", ProcessEnginesFilter.class, "/api/*", "/app/*");
             registerFilter("AuthenticationFilter", AuthenticationFilter.class, "/api/*", "/app/*");
-            // TODO add Security Filter
-            // TODO add CsrfPreventionFilter
-            // TODO configure headerSecurityProperties
-            registerFilter("HttpHeaderSecurityFilter", HttpHeaderSecurityFilter.class, "/api/*", "/app/*");
+            registerFilter("SecurityFilter", WebappSecurityFilter.class, singletonMap("configFile","/securityFilterRules.json"), "/api/*", "/app/*");
```

Comment:
what is "securityFilterRules.json" ?

Reply:
It is a file which is included in the webapps dependency: see https://github.com/camunda/camunda-bpm-platform/blob/7c5bf37307d3eeac3aee5724b6e4669a9992eaba/webapps/src/main/webapp/WEB-INF/securityFilterRules.json